### PR TITLE
Add site notices to data explorer

### PR DESCRIPTION
**Pull request by michaelglenister**
Issue: https://trello.com/c/cOcFTbQS/525-show-site-notices-on-data-explorer-site-incl-docs-so-that-users-who-somehow-land-there-know-that-it-isnt-official-data-est-1d

### CODE REVIEW

File: municipal_finance/templates/table.html [hunk 1]
```diff
@@ -23,7 +23,12 @@
     </div>
 
     <img src="{% static 'images/treasury-logo.png' %}" class="logo">
-    <h1><a href="/">Municipal Finance Data Tables</a></h1>
+    {% if site_notices %}
+      <h3><a href="/">Municipal Finance Data Tables</a></h3>
```
Comment (paulmwatson) [inline municipal_finance/templates/table.html:27]: This seems to be the top level heading on the page so it should use h1 not change to h3. Note CSS h3 styling above too.

File: municipal_finance/templates/docs.html [hunk 2]
```diff
@@ -34,6 +34,7 @@
 
 <div class="page-wrapper">
   <div class="content">
+    {% for notice in site_notices %}<h3>{{ notice.content | safe }}</h3>{% endfor %}
```
Comment (paulmwatson) [inline municipal_finance/templates/docs.html:37]: Let's not use a h3 for notices as they are not structurally headings. A p with a class should suffice e.g. `<p class="site-notice">....</p>`

File: municipal_finance/templates/table.html (hunk 1 above)
Comment (michaelglenister) [reply municipal_finance/templates/table.html]: Changed the heading tag and corresponding CSS

File: municipal_finance/templates/docs.html (hunk 2 above)
Comment (michaelglenister) [reply municipal_finance/templates/docs.html]: Same here, changed to div with it's own CSS

File: municipal_finance/templates/layout_data.html [hunk 3]
```diff
@@ -16,6 +16,7 @@
     <h1><a href="/">Municipal Finance Data</a></h1>
   </div>
 </header>
+{% for notice in site_notices %}<h4>{{ notice.content | safe }}</h4>{% endfor %}
```
[the remark this answers is not in the document]
Comment (michaelglenister) [reply municipal_finance/templates/layout_data.html]: Changed this too and CSS

File: municipal_finance/templates/table.html [hunk 4]
```diff
@@ -23,7 +23,13 @@
     </div>
 
     <img src="{% static 'images/treasury-logo.png' %}" class="logo">
-    <h1><a href="/">Municipal Finance Data Tables</a></h1>
+    {% if site_notices %}
+        <h1 class="site-notice"><a href="/">Municipal Finance Data Tables</a></h1>
+        {% for notice in site_notices %}<h1 class="site-notice">{{ notice.content | safe }}</h1>{% endfor %}
```
Comment (paulmwatson) [inline municipal_finance/templates/table.html:28]: `h1` used for site notice here, other changes are to use a `div`.

**Comment by paulmwatson** [comment]
Also looking at the code again I think the repeated blocks of `{% for notice in site_notices %}` can be DRYed up into a single file/template to include wherever `site-notice` is needed. I'd also like to see the CSS specified just once, not sure why it needs different styling for different places. Is the current code base a bit inconsistent in styling?

**Pull request merged**